.gitignore

```diff
@@ -32,3 +32,6 @@ out/
 .externalNativeBuild/
 .navigation/
 captures/
+
+# Config for the official ODK Collect build
```

Comment:
Maybe add a little bit of context here and say that anyone who does Play Store releases should have that folder with analytics keys? It would also be useful to have that information in the releases part of the README.